### DISCUSSION TITLE
Is there a support for CloudWatch log groups?

### DISCUSSION THREAD

**Discussion by teemuniiranen** [aws-solutions/centralized-logging-with-opensearch#178] [General]
I don't see CloudWatch Logs in the Service Log options but I undestand that it's used in some pipeline configurations. Can you create pure Log Group subscriptions with Centralized Logging or do I need to handle these manually from member accounts with "Create Amazon OpenSearch Service subscription filter" functionality?

**Comment by JoeShi** [comment]
I have created a feature request based on this discussion. https://github.com/aws-solutions/centralized-logging-with-opensearch/issues/179. Yes. currently the feature is not supported yet. We are going to make it available in future versions.
